test/pylib/cpp/ldap/prepare_instance.py

```diff
@@ -269,7 +269,8 @@ def setup(project_root: Path, port: int, instance_root: Path, byte_limit: int):
     proxy_name = 'p{}'.format(port)
     subprocess.check_output(
         ['toxiproxy-cli', 'c', proxy_name, '--listen', 'localhost:{}'.format(port + 2), '--upstream',
-            'localhost:{}'.format(port)])
+            'localhost:{}'.format(port)],
+        stderr=subprocess.STDOUT)
```

Comment:
This file will be refactored and moved in https://github.com/scylladb/scylladb/pull/23235. I can add additional logic for gathering toxiproxy logs in a separate file.

Reply:
@xtrey if you plan to capture the stderr and stdout in the logging message in your PR, i will close mine in favor of yours -- just wanted to understand the root cause of the test failure ASAP. please let me know.

Reply:
I've added logging for toxiproxy in my PR.